queries/vrl/highlights.scm

```diff
@@ -0,0 +1,118 @@
+(comment) @comment
+
+[
+  (null)
+  (boolean)
+] @constant.builtin
```

Comment:
```suggestion
(null) @constant.builtin
```

Reply:
and `(boolean) @boolean`?

Reply:
It's duplicated below (along with "true" and "false").